coderadar-rest/src/main/java/io/reflectoring/coderadar/rest/query/GetCommitsInProjectController.java

```diff
@@ -24,12 +25,18 @@ public GetCommitsInProjectController(GetCommitsInProjectUseCase getCommitsInProj
 
   @GetMapping(path = "/projects/{projectId}/commits", produces = MediaType.APPLICATION_JSON_VALUE)
   public ResponseEntity<List<GetCommitResponse>> listCommits(@PathVariable("projectId") Long projectId) {
-    return new ResponseEntity<>(getCommitsInProjectUseCase.get(projectId).stream().map(commit -> new GetCommitResponse()
-            .setName(commit.getName())
-            .setAnalyzed(commit.isAnalyzed())
-            .setAuthor(commit.getAuthor())
-            .setComment(commit.getComment())
-            .setTimestamp(commit.getTimestamp()))
-            .collect(Collectors.toList()), HttpStatus.OK);
+    List<Commit> commits = getCommitsInProjectUseCase.get(projectId);
+    int commitsSize = commits.size();
+    List<GetCommitResponse> responses = new ArrayList<>(commitsSize);
+    for(int i = 0; i < commitsSize; ++i){
```

Comment:
Is this noticeably faster than a normal for-each loop?

Reply:
Should be, but not noticeably. I've changed this back to a for-each loop.